go/libkb/merkle_client.go

```diff
@@ -738,10 +738,13 @@ func (mc *MerkleClient) lookupPathAndSkipSequenceHelper(m MetaContext, q HTTPArg
 	}
 
 	apiRes, err = m.G().API.Get(m, APIArg{
-		Endpoint:       "merkle/path",
-		SessionType:    APISessionTypeNONE,
-		Args:           q,
-		AppStatusCodes: []int{SCOk, SCNotFound, SCDeleted},
+		Endpoint:        "merkle/path",
+		SessionType:     APISessionTypeNONE,
+		Args:            q,
+		AppStatusCodes:  []int{SCOk, SCNotFound, SCDeleted},
+		RetryCount:      3,
+		InitialTimeout:  4 * time.Second,
+		RetryMultiplier: 1.1,
```

Comment:
what's multiplier for?

Reply:
it changes the timeouts for subsequent requests.

Reply:
yeah I just wonder since this is the only non-default one in the app why that's desirable

Reply:
I took it from "home"... whatever that is. No multipler is OK too, we mostly care about retry in sensitive situations here (for example what happened in the UI). This is most likely what we would have to do if we were to add support for poor network conditions.